spring-cloud-dataflow-apps-plugin/spring-cloud-dataflow-apps-generator-plugin/src/main/resources/template/app-pom.xml

```diff
@@ -141,46 +141,21 @@
 				{{#app.bootPluginConfiguration}}
 				<configuration>
 					{{this}}
+					{{#app.containerImage.enableMetadata}}
+					<image>
+						<name>{{app.containerImage.orgName}}/${project.artifactId}:{{app.containerImage.tag}}</name>
+					</image>
+					{{/app.containerImage.enableMetadata}}
 				</configuration>
 				{{/app.bootPluginConfiguration}}
 			</plugin>
 
-			<plugin>
-				<groupId>com.google.cloud.tools</groupId>
-				<artifactId>jib-maven-plugin</artifactId>
-				<version>3.3.0</version>
-				<configuration>
-					<from>
-						<image>{{app.containerImage.baseImage}}</image>
-					</from>
-					<to>
-						<image>{{app.containerImage.orgName}}/${project.artifactId}:{{app.containerImage.tag}}</image>
-					</to>
-					<container>
-						<creationTime>USE_CURRENT_TIMESTAMP</creationTime>
-						<format>{{app.containerImage.format}}</format>
-						{{#app.containerImage.enableMetadata}}
-						<labels>
-							<org.springframework.cloud.dataflow.spring-configuration-metadata.json>
-								${org.springframework.cloud.dataflow.spring.configuration.metadata.json}
-							</org.springframework.cloud.dataflow.spring-configuration-metadata.json>
-							<application.name>${project.artifactId}</application.name>
-							<application.version>${project.version}</application.version>
-						</labels>
-						{{/app.containerImage.enableMetadata}}
-					</container>
-				</configuration>
-			</plugin>
-
 			{{#app.containerImage.enableMetadata}}
-			<!--
-			 Use of properties-maven-plugin to load the encoded metadata properties created by the jib-maven-plugin
-			 Use: Use: ./mvnw clean install jib:dockerBuild or ./mvnw clean install jib:build
-			-->
+
 			<plugin>
 				<groupId>org.codehaus.mojo</groupId>
 				<artifactId>properties-maven-plugin</artifactId>
-				<version>1.0.0</version>
+				<version>1.2.1</version>
```

Comment:
Please put each of the version updates is a separate commit. In this PR is fine, but lets keep them separate for history sake (and be sure we do not squash - but rather rebase and merge).

Reply:
Done